### PR TITLE
Add helper to write good first line in CL

### PR DESCRIPTION
**Pull request by musq**
Inspiration - https://gist.github.com/adeekshith/cd4c95a064977cdc6c50

### CODE REVIEW

File: review/developer/cl-descriptions.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ informative enough that they don't have to read your CL or its whole description
 just to get a general idea of what your CL actually *did*.
 
 By tradition, the first line of a CL description is a complete sentence, written
-as though it were an order (an imperative sentence). For example, say
+as though it were an order (an imperative sentence). Think of the first line as
+a response to - "*The changes in this CL will ...*". For example, say
```
Comment (adambender) [inline review/developer/cl-descriptions.md]: I like the phrasing, but have just one nit: It isn't really a 'response' to "*The changes in this CL will ...*", rather it completes the phrase. 

IOW 

Think of the first line as a way to complete the phrase: "*The changes in this CL will ...*"

Comment (musq) [reply review/developer/cl-descriptions.md]: Sounds good. I'll change the wording as suggested.

**Comment by musq** [comment]
Hi @adambender, could you please look into this.

**Comment by adambender** [comment]
Whoops sorry, lost track of this one. Lemme take another look.

**Comment by adambender** [comment]
Hey there, I ran this change by a couple folks who maintain this document internal to Google and we we concluded that it wasn't adding much clarifying value, unfortunately that means I wont be accepting the change. Again, my apologies for taking so long to get back to this PR, I just lost is in the shuffle of the end of the year.

**Pull request closed without merging**